From 69fe4740da12f9cb31903e5a514761d1c3ce8227 Mon Sep 17 00:00:00 2001
From: Fugang Duan <b38611@freescale.com>
Date: Fri, 18 Jul 2014 10:09:59 +0800
Subject: [PATCH 0454/1074] ENGR00323325 ARM: dts: imx6sx: add uart6 device
 node for dtsi

Add uart6 device node in imx6sx dtsi file.

Signed-off-by: Fugang Duan <B38611@freescale.com>
[Original patch taken from git://git.freescale.com/imx/linux-2.6-imx.git]
Signed-off-by: Biyao Zhai <biyao.zhai@windriver.com>
---
 arch/arm/boot/dts/imx6sx.dtsi |   12 ++++++++++++
 1 files changed, 12 insertions(+), 0 deletions(-)

diff --git a/arch/arm/boot/dts/imx6sx.dtsi b/arch/arm/boot/dts/imx6sx.dtsi
index b008049..4466b05 100644
--- a/arch/arm/boot/dts/imx6sx.dtsi
+++ b/arch/arm/boot/dts/imx6sx.dtsi
@@ -37,6 +37,7 @@
 		serial2 = &uart3;
 		serial3 = &uart4;
 		serial4 = &uart5;
+		serial5 = &uart6;
 		usbphy0 = &usbphy1;
 		usbphy1 = &usbphy2;
 	};
@@ -953,6 +954,17 @@
 				interrupts = <0 99 0x04>;
 			};
 
+			uart6: serial@022a0000 {
+				compatible = "fsl,imx6q-uart", "fsl,imx21-uart";
+				reg = <0x022a0000 0x4000>;
+				interrupts = <0 17 0x04>;
+				clocks = <&clks IMX6SX_CLK_UART_IPG>, <&clks IMX6SX_CLK_UART_SERIAL>;
+				clock-names = "ipg", "per";
+				dmas = <&sdma 0 4 0>, <&sdma 47 4 0>;
+				dma-names = "rx", "tx";
+				status = "disabled";
+			};
+
 			mccdemo: mccdemo{
 				compatible = "fsl,imx6sx-mcc-demo";
 				status = "okay";
-- 
1.7.5.4

